### PR TITLE
Update contribution and release guidelines

### PR DESCRIPTION
**Pull request by i-like-robots**
I've tried to cover the differences between patch and minor/major releases, which branches to raise pull requests against, and how to update the changelog.

I have also made a few updates to the contribution guide to include more information about testing.

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -79,11 +79,13 @@ Please do! All of the code in Page Kit is peer-reviewed by members of the FT cus
 
 This project follows a workflow designed around project releases. It is less strict than [Gitflow] but we encourage the separation of stable, development, and experimental branches in order to follow a scheduled release cycle.
 
-- The `master` branch is for the current stable release. Bugfixes are merged into this branch.
-- The `development` branch is for upcoming major or minor releases. This branch tracks `master` and new features are merged into it.
+- The `release-v` branches are for old releases which still be in production. Important bug fixes should be [cherry-picked] from the `master` branch as required.
```
Comment (pixelandpage) [inline contribution.md]: I'm not sure I understand what a `release-v` branch is 🤔 Do we have an example in production?

Comment (i-like-robots) [reply contribution.md]: Yes, I'm referring to these:

https://github.com/Financial-Times/dotcom-page-kit/tree/release-v0.3.x
https://github.com/Financial-Times/dotcom-page-kit/tree/release-v0.4.x

Would rewording this to `release-vX.X.X` help clarify this?

Comment (pixelandpage) [reply contribution.md]: Maybe my brain is still warming up to the New Year but I'm still not sure what these are. We don't have any releases prefixed with `release-` do we? So does this apply to any v0.3.x or v0.4.x release or is there something special about a `release-v` branch that I haven't understood? https://github.com/Financial-Times/dotcom-page-kit/tags

Comment (i-like-robots) [reply contribution.md]: Imagine we have apps in production running v1.0.0 and v2.0.0

If we needed to fix an urgent bug we would probably raise a PR against the master branch and make a patch release, v2.0.1

But that fix might also be needed by the apps still running v1.0.0. Because we can't just upgrade them to v2 we'd have to copy the patch make a v1.0.1 release too.

We can do this by finding the commit hash of the last v1 release and use it to create a new branch from that point in time. That branch would be named `release-v1.x`.

Then, using `git cherry-pick` we can copy the bugfix commits from `master` onto `release-v1.x` and create a new release.

Does that make sense? Can I improve the guidelines?

Comment (pixelandpage) [reply contribution.md]: Ok, I'm with you now! The current working makes a lot more sense now. I was definitely overcomplicating this in my head

Comment (i-like-robots) [reply contribution.md]: I've added a new section which covers this so it's not lost in this thread.
